app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if (parts[0] == "mv"
+       and parts[1] != parts[2]
+       and len(parts) == 3):
```

Comment:
and works as lazy conditions. It works to all true of first false, in this case parts[0] can raise error if parts is empty list
```suggestion
    if (
       len(parts) == 3
       and parts[0] == "mv"
       and parts[1] != parts[2]
   ):
```